debian/control

```diff
@@ -3,7 +3,7 @@ Section: net
 Priority: optional
 Maintainer: Wilmer van der Gaast <wilmer@gaast.net>
 Uploaders: Jelmer Vernooij <jelmer@debian.org>
-Standards-Version: 3.9.5
+Standards-Version: 3.9.6
```

Comment:
This should be 3.9.8 now, as two more revisions have come out since this MP was submitted.